### ISSUE TITLE
Markers move from left-top/right-bottom corner on zoom in/out. GoogleUpdate broken 

### ISSUE THREAD

**Issue by tyfoan** [google-map-react/google-map-react#630]
Any example of map with markers I checked has this behavior: when you zoom in/zoom our, markers disappear, then appear in the top left or bottom right corner and then quickly get to their position. 

This problem was fixed once. We had to change version in bootstrapURLKeys: { v: 3.30 }, but now it come back after google update.
Any thoughts to solve this?

Example:
https://jsbin.com/roqutisoqu/1/edit?js,console,output

**Comment by jarodhanko** [comment]
Updating your version (which is very outdated) of google-map-react fixes this problem.

You are using 
<script src="https://npmcdn.com/google-map-react@0.14.6/dist/GoogleMapReact.js"></script>

Updating to 
<script src="https://npmcdn.com/google-map-react@1.0.6/dist/GoogleMapReact.js"></script>

fixes it for me.

**Comment by itsmichaeldiego** [comment]
As @jarodhanko stated, your version is very old, upgrading the version to latest: `1.0.6` fixes the problem.

See: https://jsbin.com/dufuzozapa/edit?html,js,console,output

**Comment by itsmichaeldiego** [comment]
@tyfoan Let me know if that helps

**Comment by nicoqh** [comment]
If this is due to an outdated version, do the examples need to be updated as well? The markers seem to jump when zooming.

**Comment by itsmichaeldiego** [comment]
@nicoqh That is already accounted in #549, I've been working in new examples here: https://github.com/itsmichaeldiego/google-map-react-examples, let me know if you want to help! A lot of help is needed.

**Comment by itsmichaeldiego** [comment]
@nicoqh No worries! Let me know if you need any help.

**Comment by saikatharryc** [comment]
@itsmichaeldiego 
is this issue solved? cause using 1.1.2 and still in this you can see marker is moving, when zooming out.
https://jsbin.com/tucukum/edit?html,css,js,console,output

**Comment by saikatharryc** [comment]
https://jsbin.com/tucukum/edit?html,css,js,console,output
@itsmichaeldiego  is there any issue with my code? 
cause specifically when i try to zoom out, the marker position becomes wrong at that time i guess.
it will be awesome, if anyone could help me out with this.

**Comment by itsmichaeldiego** [comment]
@saikatharryc Yup, take a look at this https://github.com/google-map-react/google-map-react/blob/master/API.md#positioning-a-marker and let me know if it works

**Comment by saikatharryc** [comment]
@itsmichaeldiego  doesnot seem to resolve the issue.
have a look here once  may be: https://jsbin.com/tucukum/edit?html,css,js,console,output

**Comment by jarodhanko** [comment]
@saikatharryc You need to make sure that the car is centered over the position on the map.  Looking at your sample, it looks like the marker is several pixels to the right of the actual position.  

Try the following for marker:
```
const Marker = ({ metaData, deg }) => (
    <div style={{ position: "absolute", transform: "translate(-50%, -50%)" }}>
       
        {metaData == "cartop" && (
            <svg width="40" height="50" id="Capa_1">
                <g xmlns="http://www.w3.org/2000/svg">
                    <path d="M29.395,0H17.636c-3.117,0-5.643,3.467-5.643,6.584v34.804c0,3.116,2.526,5.644,5.643,5.644h11.759   c3.116,0,5.644-2.527,5.644-5.644V6.584C35.037,3.467,32.511,0,29.395,0z M34.05,14.188v11.665l-2.729,0.351v-4.806L34.05,14.188z    M32.618,10.773c-1.016,3.9-2.219,8.51-2.219,8.51H16.631l-2.222-8.51C14.41,10.773,23.293,7.755,32.618,10.773z M15.741,21.713   v4.492l-2.73-0.349V14.502L15.741,21.713z M13.011,37.938V27.579l2.73,0.343v8.196L13.011,37.938z M14.568,40.882l2.218-3.336   h13.771l2.219,3.336H14.568z M31.321,35.805v-7.872l2.729-0.355v10.048L31.321,35.805z" />
                </g>
            </svg>
        )}
        
    </div>
);

```

Notice it is absolutely positioned, and the icon width and height is (almost) exactly the size of the car

**Comment by saikatharryc** [comment]
@jarodhanko  thanks, this is working perfectly, now i get whats the actual issue, from your explanation.
